src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.Color;
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
```

Comment:
Consider using `Math.PI` instead of defining your own `PI` constant. `Math.PI` provides a more accurate value of π and is a standard practice in Java.